### PR TITLE
Change api time return null instead of 0

### PR DESCRIPTION
**Pull request by debuggy**

### CODE REVIEW

**Comment by yqwang-ms** [comment]
Create a common function for this?
@abuccts to comment

File: src/rest-server/src/models/v2/job/k8s.js [hunk 1]
```diff
@@ -167,8 +167,8 @@ const convertFrameworkSummary = (framework) => {
       resource: 0,
     },
     retryDelayTime: framework.status.retryPolicyStatus.retryDelaySec,
```
Comment (yqwang-ms) [inline src/rest-server/src/models/v2/job/k8s.js:169]: Also fix job history part?

**Comment by yqwang-ms** [comment]
why so many other commits? Could you compact to 1 commit and force push?

**Comment by debuggy** [comment]
> 
> 
> why so many other commits? Could you compact to 1 commit and force push?

because I merge commits from fix-time-error-release branch, it will be squashed when merged

**Comment by debuggy** [comment]
> Could you compact to 1 commit and force push?

You could only take care of file changes in all commits, because in this pr only files changed matters. And I will squash all commits into one before merge into master, it equals to ```compact to 1 commit and force push```

**Pull request merged**